styles.css

```diff
@@ -0,0 +1,31 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 30px;
+  background-color: rgb(166, 166, 228);
+}
+
+#title {
+  text-align: center;
+  font-size: 40px;
+  margin: 10px auto;
+  color: rgb(66, 66, 128);
+}
+
+#description {
+  text-align: center;
+  font-size: 25px;
+  color: #fff;
+}
+
```

Comment:
- [ ] Great job posting a message asking for "changes"!, Remember that this project is about your ability to use GitHub review tools to review/comment your code or other developers in a way that is almost self explainatory. 

  If you write a message asking to the developer to fix some issues in the code he is working on, and point out that the requested changes will follow the message, we should always make sure those notes are below our initial message in `inline comments` like this one.

  You're almost there, let's give it another chance.